### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by faddat**
## Description
don't edit config.toml, just feed seeds as a flag, reducing the total number of steps and making automation easier

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [x] Updated relevant documentation (`docs/`)
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Review by shahankhatch — APPROVED** [review]
lgtm

**Review by alessio — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,35 +33,13 @@ make install
...
-
 Download Genesis, Start your Node, Check your Node Status:
 
 ```bash
 # Download genesis.json
 wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/cosmos/launch/master/genesis.json
-# Start Gaiad
-gaiad start
+# Start Gaiad and feed it seeds
+gaiad start --p2p.seeds ba3bacc714817218562f743178228f23678b2873@5.83.160.108:26656,1e63e84945837b026f596ed8ae68708783d04ad4@51.75.145.123:26656,d2d452e7c9c43fa5ef017552688de60a5c0053ee@34.245.217.163:26656,dd36969b56c740bb40bb8badd4d4c6facc35dc24@206.189.115.41:26656,a0aca8fb801c69653a290bd44872e8457f8b0982@47.99.180.54:26656,27f8dd3bdbecbef7192291083706c156e523d8e0@3.122.248.21:26656,aee0df1a660f301d456a0c2f805b372f7341e8ec@63.35.230.143:26656,7d1f660b361d6286715c098a3a171e554e9642bb@34.254.205.37:26656,fa105c2291ac4aa452552fa4835266300a8209e1@88.198.41.62:26656,bd410d4564f7e0dd9a0eb16a64c337a059e11b80@47.103.35.130:26656
```
Comment (tac0turtle) [inline README.md:42]: Should add a note this is required each time you restart the node

**Comment by tac0turtle** [comment]
oh ci is broken. Ill push a fix then we can merge this.

**Comment by tac0turtle** [comment]
@faddat could you merge main into your branch, we will be able to merge after this

**Pull request merged**